### DISCUSSION TITLE
Showing constants as single lines on X axis

### DISCUSSION THREAD

**Discussion by marco-minervino** [grafana/grafana#60296] [Dashboards, Panels & Visualizations]
Hi,
I think it would be useful to have the possibility to use the constant transformation to replicate a value (calculated from query and consisting of a single data point, at least in my case) over the timespan of the graph showed in order to see an horizontal line on the X axis.
For example with a query like:
rate(grafana_http_request_duration_seconds_count{}[$__range]) 
with type instant and by overriding by query result>transform>constant it would show instead of single points, different horizontal lines.
My use case would be to show as an horizontal line the 95th percentile value of traffic of network ports calculated over `[$__range:]`.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
